### DISCUSSION TITLE
[ci] Auto release to npm on every commit to master

### DISCUSSION THREAD

**Discussion by Johan-dutoit** [react-native-image-picker/react-native-image-picker#1520] [Ideas]
To reduce time spent on maintaining this project, it would be in everyones interest if this was automatically published on each commit to master.

This would require us to be more pedantic about Pull Requests (i.e. more testing, improve standards, etc.)

**Comment by ravirajn22** [comment]
Instead of each commit can we have a button or something like that which will release?  or simple script which will release. We can hold auto-release for now.

**Comment by Johan-dutoit** [comment]
CI sorted out.  Automatically releases and generates a changelog.
This removes one more bottleneck.
